payjoin/src/core/receive/error.rs

```diff
@@ -109,6 +109,24 @@ impl JsonReply {
         serde_json::Value::Object(map)
     }
 
+    #[cfg(feature = "v2")]
+    pub(crate) fn from_bytes(bytes: &[u8]) -> Result<Self, serde_json::Error> {
```

Comment:
These methods being feature flagged is indicative of 
1. v1 sender should also be trying to parse the error reply
2. The `JsonReply` and othe related structs should live in common src/core